.woodpecker/notification.yaml

```diff
@@ -0,0 +1,39 @@
+variables:
+  - &golang_image 'docker.io/golang:1.24'
+  - &qa_repo 'https://github.com/opencloud-eu/qa.git'
+  - &qa_repo_branch 'matrix-notification-script'
```

Comment:
this needs to be changed as soon as https://github.com/opencloud-eu/qa/pull/53 is merged